### PR TITLE
Remove DeprecationWarning from a moved p.a.z3cform.widgets import

### PR DESCRIPTION
**Pull request by jensens**

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: plone/app/dexterity/behaviors/metadata.py [hunk 1]
```diff
@@ -2,9 +2,9 @@
 from datetime import datetime
 from DateTime import DateTime
 from plone.app.dexterity import _
-from plone.app.z3cform.widget import AjaxSelectFieldWidget
-from plone.app.z3cform.widget import DatetimeFieldWidget
-from plone.app.z3cform.widget import SelectFieldWidget
+from plone.app.z3cform.widgets.datetime import DatetimeFieldWidget
+from plone.app.z3cform.widgets.select import AjaxSelectFieldWidget
+from plone.app.z3cform.widgets.select import SelectFieldWidget
```
Comment (petschki) [inline plone/app/dexterity/behaviors/metadata.py:7]: The correct import for the former `SelectFieldWidget` is now `.widgets.select.Select2FieldWidget` ... see https://github.com/plone/plone.app.z3cform/blob/master/plone/app/z3cform/widget.py#L54 ... the `SelectFieldWidget` is a bare Bootstrap5 flavored `<select />`

**Review by petschki — CHANGES_REQUESTED** [review]

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by petschki** [comment]
CI tests for 6.0 should be dropped as this change targets Plone 6.1 only

**Comment by davisagli** [comment]
@petschki If there are changes which are for 6.1 only, then a new maintenance branch needs to be created from current master and set as the source in https://github.com/plone/buildout.coredev/blob/6.0/sources.cfg /cc @mauritsvanrees 

Then something needs to happen to make mr.roboto notice the change -- is that manual or automatic @gforcada?

**Comment by petschki** [comment]
branches from `sources.cfg` are picked automatically by mr.roboto ... @jensens could you create a maintenance branch here?

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by gforcada** [comment]
@davisagli (and everyone anyway), `mr.roboto` indeed updates the branches automatically.

You can see the matrix of plone version / package / branch used in the following page:

https://jenkins.plone.org/roboto/branches

Note: red means that the package is checked out in `checkouts.cfg`

**Pull request merged**